docs/getting_started/benchmark_owner_demo.md

```diff
@@ -213,9 +211,9 @@ labels_path: mock_chexpert/labels
 ```
 
 !!! note
-    These paths are determined based on how the data preparation MLCube expects the input paths to be.
+    The paths are determined by MLCube's expected input path for data preparation.
```

Comment:
We should here specify which MLCube we are talking about:

`The paths are determined by the Data Preparator MLCUbe's expected...`